mappings/net/minecraft/world/chunk/light/SkyLightStorage.mapping

```diff
@@ -1,17 +1,28 @@
 CLASS net/minecraft/class_3569 net/minecraft/world/chunk/light/SkyLightStorage
-	FIELD field_15815 pendingSkylightUpdates Lit/unimi/dsi/fastutil/longs/LongSet;
-	FIELD field_15817 lightEnabled Lit/unimi/dsi/fastutil/longs/LongSet;
+	FIELD field_15815 skyLightSectionsToUpdate Lit/unimi/dsi/fastutil/longs/LongSet;
+	FIELD field_15816 skyLightSectionsToRemove Lit/unimi/dsi/fastutil/longs/LongSet;
```

Comment:
```suggestion
	FIELD field_15816 sectionsToRemove Lit/unimi/dsi/fastutil/longs/LongSet;
```
Only if the `sectionsToRemove` is private in `LightStorage`

Reply:
It is private in `LightStorage`, so could definitely remap; but I'm worried if it could still be confusing? 
Also, Enigma doesn't like it- if just editing it manually, it shouldn't cause other issues in the toolchain right?

Reply:
Yes, please rename this. Some stuff like codec fields has to be renamed manually for now